client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -85,10 +85,11 @@ const StockListComponent: FC = () => {
       ['locationName', { sortable: false }],
       {
         key: 'packUnit',
-        label: 'label.pack-unit',
+        label: 'label.pack',
         Cell: getPackUnitCell({
           getItemId: r => r.itemId,
           getPackSize: r => r.packSize,
+          getUnitName: r => r.item.unitName || null,
```

Comment:
might want to check the sort on this column - the results are a bit strange.

Reply:
Hmm, removed sorting for now. I think if there is ever a need to sort this, it would be to 'filter' by unit type for stock (I wonder if there would be a need for this ? and if there would be, maybe it should be filterable ?): d8b6c7f34951abffcdfadff144c1bdc919ac447d